src/service/viewport/viewport-impl.js

```diff
@@ -1127,7 +1130,12 @@ function createViewport(ampdoc) {
   let binding;
   if (ampdoc.isSingleDoc() &&
       getViewportType(ampdoc.win, viewer) == ViewportType.NATURAL_IOS_EMBED) {
-    binding = new ViewportBindingIosEmbedWrapper_(ampdoc.win);
+    if (isExperimentOn(ampdoc.win, 'ios-embed-sd') &&
+        ampdoc.win.Element.prototype.attachShadow) {
```

Comment:
This needs to be version guarded, too. iOS 10.3 does not scroll properly in this mode, but it does have `Element.prototype.attachShadow`.

Reply:
I see. Good to know. Fixed. Tests updated.